templates/cluster-template-windows.yaml

```diff
@@ -262,11 +262,8 @@ spec:
         path: C:/create-external-network.ps1
         permissions: "0744"
       - content: |
-          [System.Text.Encoding]::UTF8.GetString([System.Convert]::FromBase64String("${AZURE_SSH_PUBLIC_KEY_B64:=''}")) | Add-Content C:/ProgramData/ssh/administrators_authorized_keys
-          icacls C:/ProgramData/ssh/administrators_authorized_keys /remove "NT AUTHORITY\Authenticated Users"
```

Comment:
Is there a reason to remove updated the perms on the authorized_keys file?

I think the goal was to ensure that a pod couldn't mount c:/ProgramData/ssh/ and steal the SSH keys used to connect to the node.
Can we double check that containers cannot mount c:/users/capi/.ssh/ and access the files?

Reply:
these are only public keys so stealing them won't do much. I can verify that a pod can't drop a key there.

Reply:
see comment below https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1296#issuecomment-817014053